components/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/APIProviderImpl.java

```diff
@@ -121,14 +121,15 @@ public void updateBusinessOwner(BusinessOwner businessOwner) throws AppManagemen
 
     /**
      * Get custom properties of a given business owner.
-     * @param ownerId
+     * @param businessOwnerId
```

Comment:
Java doc comments issues and formatting issues are fixed in https://github.com/wso2/carbon-appmgt/pull/539/files